tests/models/test_dag.py

```diff
@@ -1341,3 +1344,26 @@ class DAGsubclass(DAG):
         self.assertEqual(hash(dag_eq), hash(dag))
         self.assertNotEqual(hash(dag_diff_name), hash(dag))
         self.assertNotEqual(hash(dag_subclass), hash(dag))
+
+
+class TestPerformance(unittest.TestCase):
```

Comment:
I would like to suggest to add new marker `perfomance`. Then we don't have to run this test multiple times. What do you think @mik-laj ?

Reply:
I don't think that's needed. These are very simple and small tests. In the future, when we introduce integration tests and tests for components, not just functions, we can do it. For me, this is a unit test, although it has slightly different assertions. Performance tests are tests that have metrics that can change in the environment. These tests do not change in the environment.

Reply:
I don't think that's needed. These are very simple and small tests. In the future, when we introduce integration tests and tests for components, not just functions, we can do it. For me, this is a unit test, although it has slightly different assertions. Performance tests are tests that have metrics that can change in the environment. These tests do not change in the environment.

Reply:
Work on other performance tests will start soon and then we should plan a solution on how we will mark the tests.

Reply:
Sounds good to me. 👍